server/routerlicious/packages/services/src/sessionTracker.ts

```diff
@@ -162,6 +176,22 @@ export class CollaborationSessionTracker implements ICollaborationSessionTracker
 			});
 			return;
 		}
+
+		// Update session with accumulated client metrics if provided
+		const updatedTelemetryProperties = { ...existingSession.telemetryProperties };
+		if (clientMetrics) {
+			if (clientMetrics.opCount !== undefined && clientMetrics.opCount > 0) {
+				updatedTelemetryProperties.sessionOpCount =
+					(updatedTelemetryProperties.sessionOpCount || 0) + clientMetrics.opCount;
+			}
+			if (clientMetrics.signalCount !== undefined && clientMetrics.signalCount > 0) {
+				updatedTelemetryProperties.sessionSignalCount =
+					(updatedTelemetryProperties.sessionSignalCount || 0) +
+					clientMetrics.signalCount;
+			}
```

Comment:
For this block, logic could be simplified a lot to just

```ts
if (clientMetrics) {
  updatedTelemetryProperties.sessionOpCount = (updatedTelemetryProperties.sessionOpCount ?? 0) + (clientMetrics.opCount ?? 0);
  updatedTelemetryProperties.sessionSignalCount = (updatedTelemetryProperties.sessionSignalCount ?? 0) + (clientMetrics.signalCount ?? 0);
}
```

Reply:
Addressed in latest commit